cucumber-rails.gemspec

```diff
@@ -12,23 +12,23 @@ Gem::Specification.new do |s|
 
   s.license = 'MIT'
 
-  s.add_runtime_dependency('capybara', ['>= 1.1.2', '< 3'])
+  s.add_runtime_dependency('capybara', ['>= 1.1.2', '< 4'])
   s.add_runtime_dependency('cucumber', ['>= 3.0.2', '< 4'])
-  s.add_runtime_dependency('nokogiri', '~> 1.5')
-  s.add_runtime_dependency('railties', ['>= 4', '< 5.2'])
+  s.add_runtime_dependency('nokogiri', '~> 1.8')
+  s.add_runtime_dependency('railties', ['>= 4', '< 6'])
   s.add_runtime_dependency('mime-types', ['>= 1.17', '< 4'])
 
   # Main development dependencies
   s.add_development_dependency('ammeter', ['>= 1.0.0', '!= 1.1.3'])
   s.add_development_dependency('appraisal', '>= 0.5.1')
-  s.add_development_dependency('aruba', '~> 0.14.2')
+  s.add_development_dependency('aruba', '>= 0.14.2')
   s.add_development_dependency('builder', ['>= 3.1.0', '< 4'])
   s.add_development_dependency('bundler', '>= 1.3.5')
   s.add_development_dependency('selenium-webdriver', '>= 3.4.1')
   s.add_development_dependency('database_cleaner', '>= 1.0.0')
   s.add_development_dependency('factory_girl', '>= 3.2')
```

Comment:
Consider changing this to `factory_bot` while you're here too. There will need to be some other changes made to the features (like `emulate_javascript.feature` too)

Reply:
I've just submitted a PR to your repo for this: https://github.com/gobijan/cucumber-rails/pull/2